src/app/(nextjs_migration)/components/client/PageLoad.tsx

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { useEffect } from "react";
+import { useGlean } from "../../../hooks/useGlean";
+
+interface PageLoadProps {
+  pageTitle: string;
+}
+
+// Empty component that records a page view on first load.
+export const PageLoad = ({ pageTitle }: PageLoadProps) => {
+  const { appEvents } = useGlean();
+
+  // On first load of the page, record a page view.
+  useEffect(() => {
+    if (pageTitle) {
+      appEvents.pageView.record({ label: pageTitle });
```

Comment:
You likely want to record the page path here as well (or that's pretty much the one thing you want to record)

Reply:
After seeing @rhelmer slack, we likely need to add everything that's in https://support.google.com/analytics/answer/9234069?hl=en

Reply:
Yes, I think for this PR we don't need to, Monitor team can do that as a followup - we'd like to just land something simple enough to do an A/A dry-run test of Nimbus integration.